### PR TITLE
Macos fix

### PR DESCRIPTION
**Pull request by AnasMostefaoui**
Fix an issue due to CRLF ending
Make adjustment to make it work from macOS
add down command.
Update read me

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,7 @@
 /network
 docker-compose-env.ps1
 docker-compose-env.sh
+
+# alert manager generated files.
```
Comment (smellouk) [inline .gitignore]: I think it's not necessary
You can bind the volume to another path to avoid this

File: compose/network-compose.yaml [hunk 2]
```diff
@@ -87,14 +87,14 @@ services:
...
-    
+
     #########################################
     ############## adguardhome ##############
     #########################################
     adguardhome:
         image: adguard/adguardhome
         container_name: adguardhome
-        hostname: traefik
+        hostname: adguard
```
Comment (smellouk) [inline compose/network-compose.yaml:97]: :+1:

File: .gitignore (hunk 1 above)
Comment (AnasMostefaoui) [reply .gitignore]: by default it was set in root.

Comment (smellouk) [reply .gitignore]: in monitoring compose is set to 
```
            - ../monitoring/alertmanager:/etc/alertmanager
            - ../monitoring/alertmanager:/alertmanager
```

**Review by smellouk — APPROVED** [review]

**Pull request merged**